fiftyone/core/odm/dataset.py

```diff
@@ -357,6 +380,128 @@ def _rename_paths(self, paths, new_paths):
             self._rename_path(path, new_path)
 
 
+def _make_default_sidebar_groups(sample_collection):
+    # Possible sidebar groups
+    metadata = []
+    labels = []
+    frame_labels = []
+    custom = []
+    primitives = []
+    other = []
+    unsupported = []
+
+    # Parse sample fields
+    schema = sample_collection.get_field_schema()
+    _parse_schema(
+        schema,
+        metadata,
+        labels,
+        frame_labels,
+        custom,
+        primitives,
+        other,
+        unsupported,
+    )
+
+    # Parse frame fields
+    if sample_collection._contains_videos():
+        schema = sample_collection.get_frame_field_schema()
+        _parse_schema(
+            schema,
+            metadata,
+            labels,
+            frame_labels,
+            custom,
+            primitives,
+            other,
+            unsupported,
+            frames=True,
+        )
+
+    sidebar_groups = [
+        SidebarGroupDocument(name="tags", expanded=True),
+        SidebarGroupDocument(name="label tags", expanded=True),
+        SidebarGroupDocument(name="metadata", paths=metadata, expanded=False),
```

Comment:
Not something that needs to be addressed by this PR, but I noticed all the metadata fields are prefixed with `metadata`, which causes some of the longer field names to be hidden. Since the metadata section already has a heading, is there a reason we need to display the prefix?

Reply:
Yeah that's a good observation. A couple comments:
1. This PR is duplicating server-side the clientside logic that already exists and is performed to construct default sidebar groups if none have been explicitly configured
2. The reason that the `metadata` section uses `metadata.XXX` for the individual field names is that the group name is arbitrary, so if the user renamed it to something other than `METADATA`, then they might be confused where the `XXX` field is coming from